README.md

```diff
@@ -83,6 +83,33 @@ This can also be a string accepted by the
 
 Serve files relative to `path`.
 
+##### fs
+
+Use a filesystem other than the node-js built-in `fs` module. It must support the following methods:
+
+ * stat(path, callback)
+ * createReadStream(path, options)
+
+The resulting stat object from `stat` must (at least) look like:
+
+```javascript
+{
+    mtime: Date,
+    ctime: Date,
+    ino: Number,
+    size: Number
+}
+```
+
+The resulting readable stream must (at least) support the options:
```

Comment:
To me, if this is going to be stated, we need to explain what these options mean and how someone can actually implement them. Otherwise, it's going to lead to issues asking about it.